components/list/list-item-checkbox-mixin.js

```diff
@@ -81,7 +81,7 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(L
 		super.connectedCallback();
 		if (this.selectable) {
 			if (!this.key) console.warn('ListItemCheckboxMixin requires a key.');
-			if (!this.label || this.label.length === 0) console.warn('ListItemCheckboxMixin requires a label.');
+			if (!this.label && !this.labelledBy) console.warn('ListItemCheckboxMixin requires a label.');
```

Comment:
I am guessing we will want to change this to throw an error?  I needed to update this to alternatively check for labelled-by.

Reply:
I was hoping the mixin code I added would either catch this case (making this warning unnecessary) or be able to be modified slightly to catch it.

Reply:
It's fine if you want to exclude this from these changes... I can make this change separately.  It's not specific to the labelled-mixin, but is related.

Reply:
This scenario is that the item is selectable, but they did not provide a label or a reference to a label.  So currently,   I think the `LabelledMixin` mixin would only catch the scenario where they assigned a `labelled-by` value but it is not valid.